orderer/common/server/main.go

```diff
@@ -331,6 +364,11 @@ func selectClusterBootBlock(bootstrapBlock, sysChanLastConfig *cb.Block) *cb.Blo
 		return bootstrapBlock
 	}
 
+	if bootstrapBlock == nil {
+		logger.Info("Selected system channel last config block, because bootstrap block is nil")
```

Comment:
Any reason to log this at info and not debug like the message directly above does?

Reply:
Changed